source/getEntityTypescriptPostgres.ts

```diff
@@ -281,10 +281,11 @@ async function getEntityTypescriptPostgres(
 			case "USER-DEFINED":
 				decorator = `@Field`;
 				decoratorArgsValueType += `() => ${toPascalCase(udt_name)}`;
-				type = null;
 				if (column_default === null) {
+					type = `${toPascalCase(udt_name)} | null`;
```

Comment:
This is the case only when we have also `allowNull: true`

I have something like: 
```
@Field(() => QuoteLevelEnum)
level: QuoteLevelEnum | null = null
```

Another point, should we add the `?`
like: 
```
level?: QuoteLevelEnum | null = null
```

Reply:
the `?` is required by ts